### ISSUE TITLE
Enhance local rest scorer to support HTTPs natively

### ISSUE THREAD

**Issue by hemenkapadia** [h2oai/dai-deployment-templates#244]
Support ticket  - https://support.h2o.ai/a/tickets/100029

Customer needs native HTTPS support in rest scorer. We provided a solution fronting scorer with nginix for SSL termination but the customer is seeking a native solution.

**Comment by somanathghalimath** [comment]
Hi @mmalohlava 
I did some research and came up with a way to natively support the https requirement for our local rest scorer without modification to the scorer code itself. Basically I found out that java uses its own trust store to maintain the certificates/secret keys etc in the java key store. The would mean that we have to convert the SSL certificate and the private key into pk12 format and store them in a keystore on the java classpath.

Below are the steps that I followed
1. Convert certificate and key into pk12 keystore (customer would need to have openssl package installed)
`openssl pkcs12 -export -in server_intermediate_root_chain.pem -inkey server_wildcard-key.pem -name scorer -out scorer.p12
`
2. Create a Java keystore and import the pk12 created above (keytool is a utility provided within the JDK)
`keytool -importkeystore -deststorepass password -destkeystore scorer.jks -srckeystore scorer.p12 -srcstoretype PKCS12
`
3. Run the scorer by placing this keystore in its classpath, add sudo to run on port 443.
`sudo java -cp local-rest-scorer-1.1.2-boot.jar:scorer.jks -Dmojo.path=iris.mojo -Dai.h2o.mojos.runtime.license.file=license.sig -Dserver.port=443 -Dserver.ssl.key-store=classpath:scorer.jks -Dserver.ssl.enabled=true -Dloader.main=ai.h2o.mojos.deploy.local.rest.ScorerApplication  org.springframework.boot.loader.PropertiesLauncher
`
This enables the SSL on scoring natively from the rest scorer itself.

`curl -X POST -H "Content-Type: application/json" -d @test.json https://scorer.172.19.0.1.sslip.io/model/score
`_result - {"id":"313d9222-f045-11eb-a7a7-6ac0830a48eb","fields":["Species.Iris-setosa","Species.Iris-versicolor","Species.Iris-virginica"],"score":[["0.9735661183120918","0.017293803620843255","0.009140078067064976"]]}_

While this can help us enable ssl without any changes to the code while the cons are that some additional steps still need to be taken by customer to get this working. 
Requesting your comments on this approach.

cc: @hemenkapadia @nkpng2k
